subsys/bluetooth/controller/Kconfig

```
@@ -527,11 +533,19 @@ config BT_CTLR_ADV_ISO_SET
 	help
 	  Maximum supported advertising sets.
 
-config BT_CTLR_ADV_DATA_LEN_MAX
-	int "Maximum Advertising Data Length"
-	range 31 1650
+config BT_CTLR_ADV_ISO_PDU_LEN_MAX
+	int "Maximum Broadcast Isochronous Channel PDU Length"
+	depends on BT_CTLR_ADV_ISO
+	range 27 251
 	help
-	  Maximum Extended Advertising Data Length.
+	  Maximum Broadcast Isochronous Channel PDU Length.
+
+config BT_CTLR_SYNC_ISO_PDU_LEN_MAX
+	int "Maximum Synchronized Isochronous Channel PDU Length"
+	depends on BT_CTLR_SYNC_ISO
+	range 27 251
```

I don't think it is permissible to restrict the PDU length on the receiver side, at least I did not find anything in the specification that would indicate that. If the host sends the command to synchronize with a BIS, I do not see how the controller would reject based on the Max_PDU parameter. Am I overlooking something?

---

An application desiring to synchronize with a broadcaster supported in its eco-system can use the `HCI_LE_BIGInfo_Advertising_Report ` containing the `Max_PDU`. Mostly, a source and sink would be paired and tuned for optimal PDU sizes. Here the intention to exposed the Kconfig is to allow build applications with reduced Rx buffer sizes.

---

Okay. But at least when configuring with HCI_RAW, this _must_ be set to 251 to be able to pass qualification. There should also be default values provided. For the ISO PDUs, I propose a default of 251; I am not sure what a suitable default for the advertising data length would be.

---

I have added the default 251 when building with BT_HCI_RAW.

---

If Kconfig allows it, I suggest the following:
```suggestion
	range 27 251 if !BT_HCI_RAW
	range 251 251 if BT_HCI_RAW
	default 251
```
This should ensure that there is always a value assigned and that HCI_RAW can only be configured to conform to the specification.

---

Done. The explicit `!BT_HCI_RAW` is not needed.